### ISSUE TITLE
[WEBSITE] - Fix Product Section

### ISSUE THREAD

**Issue by RavindraP04** [Clueless-Community/seamless-ui#2238]
# Fix Product Section mobile screen (360px)
- Fix Text Size 
- Give px as shown below 

## Instructions 
- You can get the code of the website in `website-dev` branch 
- After making changes, Push the changes to `website-dev` branch 
- You will find the code of this components [here](https://github.com/Clueless-Community/seamless-ui/blob/website-dev/components/Product.tsx)
- For the mobile screen change the size of image in the card at 360px 👇

### Currently This is how it looks in mobile 
<img align="left" width="350" src="https://github.com/Clueless-Community/seamless-ui/assets/98400348/074d1da2-b618-4082-a372-51f711a3b5ea" />

<img align="right" width="350" src="https://github.com/Clueless-Community/seamless-ui/assets/98400348/cba41980-d4c5-4f6a-8cee-90d1fe1edebc" />

**Comment by Lakhaninawaz** [comment]
@RavindraP04 can you say how can run this project in locally?

**Comment by RavindraP04** [comment]
> @RavindraP04 can you say how can run this project in locally?

@Lakhaninawaz 

- Fork the repo then git clone 
- to work on the website, switch branch to `website-dev` 
- run `pnpm i` to install all dependencies 
- run `npm run dev` to start localhost

**Comment by codescalper** [comment]
![image](https://github.com/Clueless-Community/seamless-ui/assets/92156721/e1de7f1b-e061-414f-b5c4-14b6a0ced61b)
will this work? added padding left and right (pl and pr) and reduced the text size of the heading **A product by clueless
**

**Comment by Lakhaninawaz** [comment]
Ok done it @RavindraP04

**Comment by RavindraP04** [comment]
@codescalper Looks good! Make the PR

**Comment by Lakhaninawaz** [comment]
@RavindraP04 can you help not run code in my laptop?

npm run dev is not run
and what is all depedencies

**Comment by Lakhaninawaz** [comment]
all dependencies done but not run `npm run dev`

**Comment by RavindraP04** [comment]
> all dependencies done but not run `npm run dev`

Try to start live server with `pnpm dev`

**Comment by codescalper** [comment]
I've made the PR @RavindraP04

**Comment by RavindraP04** [comment]
> I've made the PR @RavindraP04

@codescalper 
Yes, Checked it, Please see, I have mentioned some points in that PR
